server/jobs/kube2store/slice-event-handler.go

```diff
@@ -51,15 +51,10 @@ func (h sliceEventHandler) OnAdd(obj interface{}) {
 			Namespace:   eps.Namespace,
 			ServiceName: serviceName,
 			SourceName:  eps.Name,
-			Topology:    sliceEndpoint.Topology,
```

Comment:
Can we remove that field from the grpc API @mcluseau? 

https://github.com/kubernetes-sigs/kpng/blob/master/api/localnetv1/services.proto#L168

Reply:
we have to assess the use of topology keys, ie for local endpoint detection, I don't have that in brain-cache

Reply:
But that field is removed, I don't have that in brain-cache either

Reply:
yes but we can't just delete the line in reaction :) ie, topology is used to detect local endpoints. I'll check that.